<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="25279" started-at="2014-06-11T19:12:17Z" finished-at="2014-06-11T19:12:43Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="25279" started-at="2014-06-11T19:12:17Z" finished-at="2014-06-11T19:12:43Z">
      <class name="com.example.tests.ContactCreationTests">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.ContactCreationTests@2d6d8735]" name="setUp" is-config="true" duration-ms="7401" started-at="2014-06-11T22:12:17Z" finished-at="2014-06-11T22:12:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testContactCreationWithValidData(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactCreationTests@2d6d8735]" name="testContactCreationWithValidData" duration-ms="3700" started-at="2014-06-11T22:12:25Z" data-provider="randomValidContactGenerator" finished-at="2014-06-11T22:12:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [first_name=test-695446020, last_name=test-444430239, home_phone=, email=]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContactCreationWithValidData -->
        <test-method status="FAIL" signature="testContactCreationWithValidData(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactCreationTests@2d6d8735]" name="testContactCreationWithValidData" duration-ms="3261" started-at="2014-06-11T22:12:28Z" data-provider="randomValidContactGenerator" finished-at="2014-06-11T22:12:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [first_name=test-313009325, last_name=test497718685, home_phone=358750420, email=test-2098242072]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Lists differ at element [17]: ContactData [first_name=test-313009325, last_name=test497718685, home_phone=358750420, email=test-2098242072] != ContactData [first_name=test1878025546, last_name=test330486432, home_phone=1523084472, email=test160940399] expected [ContactData [first_name=test-313009325, last_name=test497718685, home_phone=358750420, email=test-2098242072]] but found [ContactData [first_name=test1878025546, last_name=test330486432, home_phone=1523084472, email=test160940399]]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Lists differ at element [17]: ContactData [first_name=test-313009325, last_name=test497718685, home_phone=358750420, email=test-2098242072] != ContactData [first_name=test1878025546, last_name=test330486432, home_phone=1523084472, email=test160940399] expected [ContactData [first_name=test-313009325, last_name=test497718685, home_phone=358750420, email=test-2098242072]] but found [ContactData [first_name=test1878025546, last_name=test330486432, home_phone=1523084472, email=test160940399]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:549)
	at org.testng.Assert.assertEquals(Assert.java:514)
	at com.example.tests.ContactCreationTests.testContactCreationWithValidData(ContactCreationTests.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContactCreationWithValidData -->
        <test-method status="PASS" signature="testContactCreationWithValidData(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactCreationTests@2d6d8735]" name="testContactCreationWithValidData" duration-ms="3497" started-at="2014-06-11T22:12:32Z" data-provider="randomValidContactGenerator" finished-at="2014-06-11T22:12:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [first_name=test-1278042511, last_name=test-1571226224, home_phone=2092021013, email=]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContactCreationWithValidData -->
        <test-method status="PASS" signature="testContactCreationWithValidData(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactCreationTests@2d6d8735]" name="testContactCreationWithValidData" duration-ms="3566" started-at="2014-06-11T22:12:35Z" data-provider="randomValidContactGenerator" finished-at="2014-06-11T22:12:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [first_name=test235985278, last_name=, home_phone=677066802, email=test-16768047]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContactCreationWithValidData -->
        <test-method status="PASS" signature="testContactCreationWithValidData(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactCreationTests@2d6d8735]" name="testContactCreationWithValidData" duration-ms="3696" started-at="2014-06-11T22:12:39Z" data-provider="randomValidContactGenerator" finished-at="2014-06-11T22:12:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [first_name=, last_name=test-1520838646, home_phone=, email=test-1257070575]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContactCreationWithValidData -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.ContactCreationTests@2d6d8735]" name="tearDown" is-config="true" duration-ms="139" started-at="2014-06-11T22:12:42Z" finished-at="2014-06-11T22:12:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.ContactCreationTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
